docs/src/tutorials/software-management.md

```diff
@@ -1,67 +1,37 @@
 
+2. Software management agent
+   The thin-edge software management agent is the one that calls the plugins.
+   You can find this process with the name `tedge_agent` in `ps` once it starts.
 
+3. Software management plugin
+   Plugins are the interfaces that call the package manager (example: apt/apt-get) to do the software management operations (Install, Remove or update)
+   You can find them in /etc/tedge/sm-plugins.
+   As of now there is only one (apt) plugin is supported.
 
-## Download and Install software management packages on the device
 
-As a prequisite, install [tedge and tedge_mapper](../howto-guides/002_installation.md) if not installed already. 
+## Enable software management feature
+ 
+Find more information about [how to install and enable software management.](../../howto-guides/0012_install_and_enable_software_management.md)
+ 
+## Managing the device software on Cumulocity cloud
 
-The thin-edge software management packages are in repository on GitHub: [thin-edge.io](https://github.com/thin-edge/thin-edge.io/releases).
-
-To download the package from github repository use the following command (use desired version):
-
-```shell
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/<package>_<version>_<arch>.deb
-```
-
-where:
-> `version` -> thin-edge.io software management components version in x.x.x format
->
-> `arch` -> architecture type (amd64, armhf)
-
-Download `tedge_apt_plugin` and `tedge_agent`
-
-```shell
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.3.0/tedge_apt_plugin_0.3.0_amd64.deb
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.3.0/tedge_agent_0.3.0_amd64.deb
-```
-
-Now the packages are downloaded, proceed to installation.
-> Note: Some OSes may require you to use `sudo` to install packages and therefore all following commands may need `sudo`.
-
-To install `tedge_apt_plugin` and `tedge_agent` on thin-edge device do:
-
-```shell
-dpkg -i tedge_apt_plugin_<version>_<arch>.deb
-dpkg -i tedge_agent<version>_<arch>.deb
-```
-
-
-
-## Start and enable the software management services
-
-The `tedge connect` will automatically start and enable the software management services.
-Find more about [how to connect thin-edge device to cloud](../howto-guides/004_connect.md)
-
-Once the thin-edge device is successfully connected to Cumulocity cloud, the **Software** option will be enabled and
-the list of softwares that are installed on the device will be visible as shown in the figure below.
-
-![Add new software](./images/start-software-management.png)
-
-
-> Note: Disconnecting thin-edge device from cloud will stop and disable the software management services.
-
-
-
-## Managing the device software on Cumulocity
-
-More info about managing the device software can be found [here](https://cumulocity.com/guides/users-guide/device-management/#managing-device-software)
+Cumulocity cloud collects the references to the software for the thin-edge devices.
```

Comment:
I don't understand that sentence.

Reply:
I would say:

Managing the software packages installed on the devices of your Cumulocity tenant is a two steps operation.

1. Populate the **software repository** with all the software packages and versions you plan to install.
2. Trigger software update operations on the devices, to install specific packages of the repository.

Reply:
updated.